pelican/plugins/test/test_show_source.py

```diff
@@ -1,63 +1,112 @@
+import collections
+import itertools
 import pathlib
+import enum
 
 import pytest
-
 import pelican
```

Comment:
The blank line that was deleted here should be retained.

Reply:
same here